pkg/mflag/flag.go

```diff
@@ -788,8 +851,12 @@ func Var(value Value, names []string, usage string) {
 // returns the error.
 func (f *FlagSet) failf(format string, a ...interface{}) error {
 	err := fmt.Errorf(format, a...)
-	fmt.Fprintln(f.out(), err)
-	f.usage()
+	fmt.Fprintln(f.Out(), err)
+	if os.Args[0] == f.name {
+		fmt.Fprintf(f.Out(), "See '%s --help'.\n", os.Args[0])
+	} else {
+		fmt.Fprintf(f.Out(), "See '%s %s --help'.\n", os.Args[0], f.name)
+	}
```

Comment:
I totally disagree.

We should follow the coreutils standard

ls --fobar
ls: unrecognized option '--fobar'
Try 'ls --help' for more information.

Only display usage if user asks for it.